code/src/components/categorySection/CategorySection.scss

```diff
@@ -0,0 +1,30 @@
+@import "@design-system";
+
+.spa-category-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: spacing(25);
+  padding-top: spacing(20);
+  background-color: var(--spa-wrapper-background-color, $white);
+  margin-top: spacing(5);
+  margin-bottom: spacing(5);
+
+  &__element {
+    display: flex;
+    justify-content: center;
+    gap: spacing(7);
+    padding: spacing(10) 0 spacing(20);
+  }
+  &__title {
+    position: absolute;
+    margin-bottom: spacing(90);
+  }
+  &__image {
+    width: 200px;
+  }
+  &__label {
+    position: absolute;
+    margin-top: spacing(52);
```

Comment:
to big margin. Find another aproach

Reply:
Fixed.  I used a different approach, although i still used absolute positioning. Its not redundant anymore.

Reply:
It is still a bad solution to use absolute positioning for title positioning. You should avoid it, to make the code more scalable and easier to debug